allauth/account/views.py

```diff
@@ -13,7 +13,9 @@
 from django.views.generic.base import TemplateResponseMixin, TemplateView, View
 from django.views.generic.edit import FormView
 
+from ..app_settings import SOCIALACCOUNT_ENABLED
 from ..exceptions import ImmediateHttpResponse
+from ..socialaccount import providers
```

Comment:
It's best not to introduce a dependency from `account` to `socialaccount` as that will result in both modules to have a cyclic dependency upon each other.